Essentials/src/messages_en.properties

```diff
@@ -326,6 +326,7 @@ north=N
 northWest=NW
 noGodWorldWarning=§4Warning\! God mode in this world disabled.
 noHomeSetPlayer=§6Player has not set a home.
+noHomeSet=§4You have no home set.
```

Comment:
This will be overwritten by crowdin, remove

Reply:
How would I go about adding a translatable message then? Sorry, new to this 😛

Reply:
The only language file you should edit is `messages.properties` with the default English translation. Any other locale files are automatically updated by Crowdin after the main properties file is edited.

Reply:
Okay, thank you.